packages/rule-typescript-config-is-valid/README.md

```diff
@@ -0,0 +1,56 @@
+# typescript-config-is-valid (`typescript-config-is-valid`)
+
+`typescript-config-is-valid` warns again providing an
+invalid TypeScript configuration file (i.e. `tsconfig.json`).
+
+## Why is this important?
+
+If you are bilding an app or a website using TypeScript, you
+need to be sure the configuration file is valid.
+
+## What does the rule check?
+
+This rule check if the typescript configuration is valid.
+To do this we are using the 
+[tsconfig schema][typescript schema] with some minor
+changes.
```

Comment:
Will be nice to have a list of what those changes are for future reference

Reply:
The change is that I'm adding the property: "additionalProperties": false, in a couple of places. I will add it to the `README.md`